CHANGES.txt

```diff
@@ -134,6 +134,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       or not if it is already in the $LIBS construction var in the
       configure context. (issue #2768).
 
+  From Nickolai Korshunov
```

Comment:
Please add your blurb to this section alphabetically ordered by your last name. (That's our practice here).